### PR TITLE
Provider place info api

### PR DESCRIPTION
**Pull request by hoangnguyen02081993**
+ Adding more function get Google Locations.
+ Adding Model GoogleLocaions.
+ Adding 4 function with place (show,add,edit,delete)
+ Adding Model request/result for place.

***commit for create table
+ Adding provider_error_place
+ Modify provider_place
***

### CODE REVIEW

File: foodcopingserver/foodcopingserver/Models/foodcopingDB.Designer.cs [hunk 1]
```diff
@@ -1,4 +1,4 @@
-﻿// T4 code generation is enabled for model 'E:\git\source\foodcoping\foodcopingserver\foodcopingserver\Models\foodcopingDB.edmx'. 
+﻿// T4 code generation is enabled for model 'D:\MMO\Src\foodcoping\foodcopingserver\foodcopingserver\Models\foodcopingDB.edmx'.
```
Comment (bxdoan) [inline foodcopingserver/foodcopingserver/Models/foodcopingDB.Designer.cs]: remove this change

File: foodcopingserver/foodcopingserver/Controllers/Provider/api/ProviderPlaceAPIController.cs [hunk 2]
```diff
@@ -0,0 +1,359 @@
...
+                    foreach (var item in PlaceList)
+                    {
+                        ProviderPlaceModel place = new ProviderPlaceModel()
+                        {
+                            placeId = item.id.ToString(),
+                            address = item.address,
+                            description = item.description,
+                            phonenumber = item.phonenumber,
+                            imagelink = item.imagelink,
+                            foodcertifyId = item.foodcertifyId,
```
Comment (bxdoan) [inline foodcopingserver/foodcopingserver/Controllers/Provider/api/ProviderPlaceAPIController.cs]: the ordinal should be the same with ordinal in table DB

File: foodcopingserver/foodcopingserver/Controllers/Provider/api/ProviderPlaceAPIController.cs [hunk 3]
```diff
@@ -0,0 +1,359 @@
...
+            else
+            {
+                result.result = 1;
+                result.ErrorCode = 1;
+                result.ErrorMessage = DataWrapper.FoocopingContext.provider_error_place.Where(e => e.Code == result.ErrorCode).FirstOrDefault().Message;
+            }
+
+            return Ok(result);
+        }
+        [Route("api/provider/delplace")]
```
Comment (bxdoan) [inline foodcopingserver/foodcopingserver/Controllers/Provider/api/ProviderPlaceAPIController.cs]: deleteplace

File: foodcopingserver/foodcopingserver/Models/foodcopingDB.cs [hunk 4]
```diff
@@ -7,3 +9,4 @@
 // </auto-generated>
 //------------------------------------------------------------------------------
```
[the remark this answers is not in the document]
Comment (hoangnguyen02081993) [reply foodcopingserver/foodcopingserver/Models/foodcopingDB.cs]: This is a auto genarated file. I don't know how to remove it.

File: foodcopingserver/foodcopingserver/Models/foodcopingDB.Designer.cs (hunk 1 above)
Comment (hoangnguyen02081993) [reply foodcopingserver/foodcopingserver/Models/foodcopingDB.Designer.cs]: This is a auto genarated file. I don't know how to remove it.

File: foodcopingserver/foodcopingserver/Controllers/Provider/api/ProviderPlaceAPIController.cs (hunk 3 above)
Comment (hoangnguyen02081993) [reply foodcopingserver/foodcopingserver/Controllers/Provider/api/ProviderPlaceAPIController.cs]: ok, thanks

**Pull request merged**